timescale-forge/disk-management.md

```diff
@@ -0,0 +1,89 @@
+# Disk management
+
+Timescale Forge contains several mechanisms for managing disk space on your
+services. There are four key tasks that Forge performs to handle disk space:
+
+1.  Detect if storage capacity begins to fill up
+1.  Notify you about the growth
+1.  Automatically activate overload protections
+1.  Allow you to return your database to a normal state
+
+This section explains what the various mechanisms are, and how to best make use
+of them.
+
+## Continuous storage monitoring
+
+Timescale Forge continuously monitors the health and resource consumption of all
+database services. You can check your health data by navigating to the `metrics`
+tab in your service dashboard. These metrics are also monitored by the Timescale
+operations team.
+
+*Insert screenshot here*
+
+If your database exceeds a storage threshold of available resources, some
+automated actions are triggered, including notifications, and preventative
+actions.
+
+## Automated user alerting
+
+When your disk usage exceeds certain thresholds, you will recieve email notifications. These notifications occur at:
+
+*   75%
+*   85%
+*   95%
+
+So that you aren't overwhelmed by automated message, the alerting thresholds use low- and high-watermarks, and only one message is sent per database per day.
+
+## Automated overload protection
+
+If your database continues to increase in size past these thresholds, automated overload protection is activated when your disk becomes 99% full. When this happens, your database is put into read-only mode, and you receive a notification on email and in the Timescale Forge console.
+
+When this happens, you can still query your database, but you cannot add any new data to it. This is to ensure that your disk does not fill up to 100%, which could prevent you from recovering the database.
```

Comment:
"When this happens" => "In read-only mode" ? 
Also, maybe more explicit about the consequences of 100% => OOM crash in the DB?  Dunno, others are more knowledgeable about what exactly happens

Reply:
```suggestion
When your database is in read-only mode, you can still query the database, but you cannot add any new data to it. This is to ensure that your disk does not fill up to 100%, which could prevent you from getting an out of memory (OOM) error.
```

I thought that failing to recover sounded scarier than OOM 😉

Reply:
maybe "prevent it from crashing due to an out of memory (OOM) error." is scarier?  @itroot is probably best to answer what would be the best way to put it.

Reply:
```suggestion
When this happens, you can still query your database, but you cannot add any new data to it. This is to ensure that your disk does not fill up to 100%, which could prevent you from crashing due to an out of memory (OOM) error.
```

I'm happy with that, if @itroot gives me a 👍